sql/testdata/drop_table

```diff
@@ -0,0 +1,30 @@
+statement ok
+CREATE TABLE a (id INT PRIMARY KEY)
+
+statement ok
+CREATE TABLE b (id INT PRIMARY KEY)
+
+query TT colnames
+SHOW TABLES FROM test
+----
+Table
+a
+b
+
+statement ok
+DROP TABLE a
+
+query T colnames
+SHOW TABLES FROM test
+----
+Table
+b
+
```

Comment:
@vivekmenezes  if i understand correctly, when the table is recreated with same name, it will still get a different table id.
So even if the new table is empty, we still don't know the old table data is deleted.
That's why I created another drop_test file and used the KV interface to test if the old data is deleted.
I will do it as you said to be safer.